ports/psoc6/main.c

```diff
@@ -44,6 +44,12 @@
 
 #define MPY_TASK_STACK_SIZE                    (4096u)
 #define MPY_TASK_PRIORITY                      (3u)
+#define BUTTON_PRESSED                         (0) // active low
+
+typedef enum {
+    BOOT_MODE_NORMAL,
+    BOOT_MODE_SAFE
+} BootMode;
```

Comment:
While we don´t have any explicit naming convention... we are commonly using across the code base "snake_case" with "_t" suffix for types. So I would do "boot_mode_t" for the enum typedef.

Reply:
fixed